crates/parcel_core/src/types/asset.rs

```diff
@@ -47,11 +48,20 @@ impl From<String> for Code {
   }
 }
 
+#[derive(
+  Default, PartialEq, Clone, Debug, Deserialize, Serialize, bincode::Encode, bincode::Decode,
```

Comment:
I think I need to enable the serde feature to use the serde traits.

Reply:
Can you update the PR to enable the feature?

Reply:
Hey not everything is supported in the serde layer, which is only a compatibility mode.

It also performs marginally worse. Because of this we could keep the derive as it's likely a better way of interfacing with bincode, even when the serde feature-flag is enabled (which still requires `Compat` wrappers around structs - https://github.com/bincode-org/bincode/tree/trunk/src/features/serde)

Reply:
Refer to - https://github.com/parcel-bundler/parcel/pull/9827/commits/c1719ad0aa14ea2cee11bdbb03667f9d468b52bd

However I've reverted it since it was crashing with https://github.com/bincode-org/bincode/blob/trunk/src/features/serde/mod.rs#L70-L73

Reply:
Okay, thanks for looking into it